src/main/java/ladder/domain/Line.java

```diff
@@ -0,0 +1,57 @@
+package ladder.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class Line {
+
+    private List<Boolean> points = new ArrayList<>();
+
+    private static final String CONSECUTIVE_TRUES_CANNOT_BE_ENTERED = "Line을 생성할 때 true가 연속으로 들어올 수 없습니다.";
+
+    public Line(List<Boolean> list) {
+        validation(list);
+        this.points = list;
+    }
+
+    private void validation(List<Boolean> list){
+        for (int i = 1; i < list.size(); i++) {
+            continuousLineValidation(list, i);
+        }
+    }
+
+    private void continuousLineValidation(List<Boolean> list, int i) {
+        if(list.get(i) == true){
+            List<Boolean> consecutiveElements = new ArrayList<>();
+            consecutiveElements.add(list.get(i - 1));
+            consecutiveElements.add(list.get(i));
+
+            inCaseOfConsecutive(consecutiveElements);
+        }
+    }
+
+    private static void inCaseOfConsecutive(List<Boolean> areConsecutiveEqual) {
+        if (areConsecutiveEqual.get(0) == areConsecutiveEqual.get(1)) {
+            throw new IllegalArgumentException(CONSECUTIVE_TRUES_CANNOT_BE_ENTERED);
+        }
+    }
```

Comment:
두 개의 boolean 을 인자로 받으면 어떨까요?
```suggestion
    private void inCaseOfConsecutive(boolean left, boolean right) {
        if (left && right) {
            throw new IllegalArgumentException(CONSECUTIVE_TRUES_CANNOT_BE_ENTERED);
        }
    }
```